web/.idea/runConfigurations/Run__malte_.xml

```diff
@@ -0,0 +1,12 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Run (malte)" type="js.build_tools.npm">
```

Comment:
Should be `Start packager (malte)` If I remember correctly.

Reply:
This is for web so I guess it is fine.

Reply:
upsi